scene/gui/rich_text_label.compat.inc

```diff
@@ -46,11 +46,18 @@ bool RichTextLabel::_remove_paragraph_bind_compat_91098(int p_paragraph) {
 	return remove_paragraph(p_paragraph, false);
 }
 
+void RichTextLabel::_set_table_column_expand_bind_compat_101482(int p_column, bool p_expand, int p_ratio) {
+	set_table_column_expand(p_column, p_expand, p_ratio, true);
+}
+
 void RichTextLabel::_bind_compatibility_methods() {
+	ClassDB::bind_compatibility_method(D_METHOD("push_font", "font", "font_size"), &RichTextLabel::push_font);
+	ClassDB::bind_compatibility_method(D_METHOD("set_table_column_expand", "column", "expand", "ratio"), &RichTextLabel::set_table_column_expand);
```

Comment:
This was missed in the review of #79053 (the compat binding system was maybe not as mature/documented yet), but this is a bit unconventional to bind compat methods directly to a non-compat method.

I'd suggest adding `push_font_bind_compat_79053` and `set_table_column_expand_bind_compat_79053` here so we keep the origin and rationale explicit.

Reply:
Added it, in this case the only difference is default argument values, method signature is exactly same, so it should not make any difference.